pootle/core/helpers.py

```diff
@@ -84,10 +84,11 @@ def get_translation_context(request):
     vfolder_pk = getattr(request, 'current_vfolder', '')
     display_priority = False
 
-    if not vfolder_pk:
-        display_priority = VirtualFolder.objects.filter(
-            units__store__pootle_path__startswith=request.pootle_path
-        ).exists()
+    if not vfolder_pk and not request.pootle_path.startswith("/projects"):
```

Comment:
im not seeing how that can be done efficiently without checking if `lang` is a valid lang code